docs/introduction.md

```diff
@@ -0,0 +1,29 @@
+---
+id: introduction
+slug: /
+---
+
+# Introduction
+
+Jellyfish is a WebRTC Media Server implemented in Elixir.
+It allows for effortless creationg of online meeting rooms.
+
+## Installation
+
+Make sure to have Elixir installed. If not, you can do it [here](https://elixir-lang.org/install.html).
+
+Clone the Jellyfish repo
+```
+git clone https://github.com/jellyfish-dev/jellyfish.git
+```
+
+Install the dependencies
+```
+mix deps.get
```

Comment:
Unfortunately, you also need to install some non-elixir dependencies and some environmental variables, see Videoroom README, `ex_dtld` README.